bleak/__init__.py

```diff
@@ -77,8 +77,19 @@
             "Requires at least Windows 10 version 0.16299 (Fall Creators Update)."
         )
 
-    from bleak.backends.dotnet.scanner import BleakScannerDotNet as BleakScanner  # noqa
-    from bleak.backends.dotnet.client import BleakClientDotNet as BleakClient  # noqa
+    py_major, py_minor, _ = platform.python_version_tuple()
+    if int(py_major) >= 3 and int(py_minor) >= 9:
```

Comment:
Yes, forgot about this. Added this comment:

```
# If the winrt package is installed, assume that the user has opted to use that backend
# instead of the pythonnet/BleakBridge implementation.
```